lib/vsc/utils/generaloption.py

```diff
@@ -725,7 +734,8 @@ def format_option_rsthelp(self, formatter=None):
                 if opt.help is not nohelp:
                     values.append(['``%s``' % formatter.option_strings[opt], formatter.expand_default(opt)])
 
-            res.extend(mk_rst_table(titles, map(list, zip(*values))))
+            columns = [list(x) for x in zip(*values)]
```

Comment:
no list comprehension, just wrap with `list` (it's already used to wrap the tuples)

Reply:
I'll make `mk_rst_table` make sure the arguments are lists rather than generator object, since that's where we're using those values multiple times, to prevent running into surprises if you re-use `mk_rst_table` in other places...